kernel-boot/modules/rdma.conf

```diff
@@ -0,0 +1,21 @@
+# These modules are loaded by the system if any RDMA devices is installed
+# iSCSI over RDMA client support
+ib_iser
+
+# iSCSI over RDMA target support
+# ib_isert
+
+# User access to RDMA verbs (supports libibverbs)
+ib_uverbs
+
+# User access to RDMA connection management (supports librdmacm)
+rdma_ucm
+
+# RDS over RDMA support
+# rds_rdma
+
+# NFS over RDMA client support
+xprtrdma
+
+# NFS over RDMA server support
+svcrdma
```

Comment:
The same is true for both nfsordma modules, they should be loaded by the NFS services when we have RDMA devices, we shouldn't be loaded them as it will trigger an autoload of these modules on systems that might not want the NFS stack enabled at all.